Sources/Purchasing/StoreKitAbstractions/PromotionalOffer.swift

```diff
@@ -122,11 +122,20 @@ extension PromotionalOffer.SignedData {
 
     @available(iOS 15.0, tvOS 15.0, watchOS 8.0, macOS 12.0, *)
     var sk2PurchaseOption: Product.PurchaseOption {
+        let signature: Data
+
+        if let decoded = Data(base64Encoded: self.signature) {
```

Comment:
I was hoping this would fix it, but the test is still failing :/

Reply:
I'm out of ideas, I'll probably have to file a Radar and pray? 🙏🏻

Reply:
⚠️ 

Looks like one of our customers tried to do this without RC, using `StoreKit` directly, and he gets the same result, so that confirms the fact that this is an Apple bug.
He's opened a TSI with Apple, so hopefully they can address this soon.

In the mean time, I'll leave this PR open. I believe the fix here will still be required (doing the `base64` decode). But once Apple fixes it on their end we can use this PR to confirm, as well as the now enabled test.

Reply:
Do you have the ticket # with Apple?

Reply:
Not sure about the original, but we reported it today as well under: FB11764024.